parmed/rosetta/pose.py

```diff
@@ -74,7 +74,7 @@ def load(pose):
                 else:
                     atom = Atom(**params)
 
-                atom.xx, atom.xy, atom.xz = tuple(at.xyz())
+                atom.xx, atom.xy, atom.xz = (at.xyz()[0], at.xyz[1], at.xyz[2])
```

Comment:
This doesn't look quite right.  why is at.xyz() called on the first, but not on the second and third?

Basically, I would have expected either:

```python
atom.xx, atom.xy, atom.xz = at.xyz()[0], at.xyz()[1], at.xyz()[2]
```
or
```python
atom.xx, atom.xy, atom.xz = at.xyz[0], at.xyz[1], at.xyz[2]
```

Am I missing something? Like a side-effect of at.xyz that turns it from a method to a property after the first call?